### PR TITLE
Change readme general

### PR DESCRIPTION
**Pull request by sleviim**

### CODE REVIEW

**Comment by danielhers** [comment]
@simonaho, let me know if you approve and we'll merge.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 ANYWAY [![Build Status](https://travis-ci.org/hasadna/anyway.png)](https://travis-ci.org/hasadna/anyway)
 ======
 
-oway.org.il - Crowd-sourced road hazard reporting website.
```
Comment (simonaho) [inline README.md]: Oway.org.il also works, we prefer it since it is org, you can leave both.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+[anyway.co.il](https://www.anyway.co.il/) - Crowd-sourced road hazard reporting website.
 
-Feel free to contribute to the project.
```
Comment (simonaho) [inline README.md]: I'm not a native English speaker, but I think it should be "to the project"

Comment (sleviim) [reply README.md]: I thought I already changed it... Probably missed it :)
will do

File: README.md (hunk 1 above)
Comment (simonaho) [reply README.md]: OK

**Pull request merged**